Makefile

```diff
@@ -59,4 +60,5 @@ security-check:
     # we fix the vulnerabilities by changing the flags on the dependency check
     # e.g.
     # -DfailBuildOnCVSS=0 or -DfailBuildOnAnyVulnerability=true
-	mvn org.owasp:dependency-check-maven:check -DassemblyAnalyzerEnabled=false
\ No newline at end of file
+	mvn org.owasp:dependency-check-maven:check -DassemblyAnalyzerEnabled=false
```

Comment:
So (I think) it's just this line that needs modifying to add the `-DfailBuildOnAnyVulnerability=true` flag at the end. No new invocation of the dependency checker is needed, so the new line you added can go. Also the three comment lines above, as they don't apply any more.

Reply:
comments removed and line edited